src/commonMain/kotlin/com/airthings/lib/logging/LogDate.kt

```diff
@@ -34,7 +34,6 @@ data class LogDate(
     val year: Int,
     val month: Int,
     val day: Int,
-    val separator: Char = '-',
```

Comment:
I think this is an important part of customizing the library. As we're boasting this as an OSS project, it's important to give users the ability and capabilities they may need, even if we don't need them at the moment.

Reply:
And while you're at it, could you sneak in KDoc for `separator` as well?

` * @param separator The character used to separate the date parts, defaults to [SEPARATOR].`

Reply:
✅